README.md

```diff
@@ -57,41 +60,103 @@ If you are interested in becoming a member of the team check out the **[Pixelogi
+  - There is currently only a 'push to master' trigger file. Other useful triggers are on PR, which can run linting,tests, and even pushed to staging, vetting the code being staged for production deployment.
+- The tag written is taken from the first line of the `.version` file. The function used splits on the colon, uses the first half using everything from the `v` forward.
+
+### Basic Workflow
+
+There is a separate Actions trigger file for each function. The Action will trigger whenever there is a change in the functions directory (e.g. `cmd/tokengen/*`). The following occurs:
+
+1. The code is checked out
+1. The version is extracted from the first line of the version file
+1. A tag is written to the merge's SHA using the version extracted from the file
+1. The config.yml file is written to disk
+1. glcoud action is loaded
+1. The deploy is run using the `.deployment` file from the function's directory.
+
+### Required Secrets
+
+The Actions configuration requires four [secrets](https://github.com/PixelogicDev/gruveebackend/settings/secrets) to be configured in GitHub repo:
+
+- **PROD_CONFIG_YAML** The configuration file used by GCP for the function's variables
+- **PROD_CONFIG_YAML_64** A base64 encoded version of `PROD_CONFIG_YAML`
+- **PROD_CLOUD_AUTH** GCP service account which can deploy to the GCP project's function. `JSON` format with no carriage returns or line feeds
+- **PROD_GCP_PROJECT_ID** The GCP project id being deployed to
+
+You may be wondering why there are both the yaml config file and the yaml config file base64 encoded. The base64 version is used to write the file used by the deploy process. The regular yaml version is used to redact all the values from the log. It's a work-around to ensure values aren't leaked in the logs. IT IS VERY IMPORTANT TO UPDATE BOTH OF THESE FILES WHENEVER THERE IS A CONFIGURATION CHANGE!
+
+### Setting up a new function for tagging and deploy on push to master
+
+Each function has it's own GitHub Actions file under `.github/workflows`. At the time of writing all of these files are triggered on **push master**. The steps for creating a file for a function is _very_ easy. There are only **three** lines to update (1, 8, 17).
+
+1. Copy `template_pushMaster.yml` file from the workflows template directory, `.github/workflow_templates/`, into the workflows directory, `.github/workflows`.
+1. Rename the file copied with the function's name. E.g. function `tokengen` would be renamed: `tokengen_pushMaster.yml`
+1. There are three lines to update in the file with the function's name: 1, 8, & 17
+    - **line 1**: Replace [function name] with the function's name (e.g. `tokengen`)
+    - **line 8**: Replace [function name] with the function's name (e.g. `tokengen`)
+    - **line 17**: Replace [function name] with the function's name (e.g. `tokengen`)
+1. That is all, GitHub Actions is now configured for the function. (after the files are pushed to master that is 😃)
```

Comment:
[nit]

Numbering here as well please!

Reply:
I always number with all 1. 's it is fully supported by markdown (which will number accordingly). And with this practice it allows you to reorder the list very easily.